### PR TITLE
[Persistability] Various checks in actions (Edit Properties, ImportFromJSON)

### LINKED ISSUE #3741: Plans folder gives false sense of editability
## Overview
Deployment specific. VIPER Plans folder does not prevent editing actions such as creating and/or renaming objects. Will get persistence errors.

## Expected Behavior
Folder shouldn't be selectable for create destination. Name should not be editable.

## Steps to Reproduce
create
1. Create any object from the Create Menu
2. Select VIPER Plans Folder to save into
3. Save

rename
1. Select VIPER Plans Folder
2. Rename a plan via the browse bar

### PR DESCRIPTION
**Pull request by jvigliotta**
### All Submissions:

Added some checks in Edit Properties and ImportFromJSON action `appliesTo` methods.

closes #3741 

* [x] Have you followed the guidelines in our [Contributing document](https://github.com/nasa/openmct/blob/master/CONTRIBUTING.md)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/nasa/openmct/pulls) for the same update/change?
* [x] Is this change backwards compatible? For example, developers won't need to change how they are calling the API or how they've extended core plugins such as Tables or Plots.

### Author Checklist

* [x] Changes address original issue?
* [ ] Unit tests included and/or updated with changes?
* [x] Command line build passes?
* [x] Has this been smoke tested?
* [x] Testing instructions included in associated issue?

### CODE REVIEW

File: src/plugins/importFromJSONAction/ImportFromJSONAction.js [hunk 1]
```diff
@@ -43,7 +43,12 @@ export default class ImportAsJSONAction {
      */
     appliesTo(objectPath) {
         const domainObject = objectPath[0];
-        if (domainObject && domainObject.locked) {
+        const locked = domainObject && domainObject.locked;
+        const persistable = this.openmct.objects.isPersistable(domainObject.identifier);
+        const TypeDefinition = this.openmct.types.get(domainObject.type).definition;
```
Comment (nikhilmandlik) [inline src/plugins/importFromJSONAction/ImportFromJSONAction.js:48]: handle 'this.openmct.types.get(domainObject.type)' is null?

Comment (jvigliotta) [reply src/plugins/importFromJSONAction/ImportFromJSONAction.js]: added the mock like you suggested, I don't think it should ver come back null, very odd.

**Review by nikhilmandlik — APPROVED** [review]
LGTM

**Comment by nikhilmandlik** [comment]
### Reviewer Checklist

* [ ] Changes appear to address issue?
* [ ] Appropriate unit tests included?
* [ ] Code style and in-line documentation are appropriate?
* [ ] Commit messages meet standards?
* [ ] Has associated issue been labelled unverified? (only applicable if this PR closes the issue)

**Pull request merged**